lib/symlink.js

```diff
@@ -6,10 +6,12 @@ var filterFunc = function(src, dest) {
   if (src.indexOf('mason.ini') > -1) {
     return false;
   } else if (fs.lstatSync(src).isFile()) {
+    if (fs.existsSync(dest)){
+      fse.removeSync(dest); 
```

Comment:
Why is this needed? Does `fs.symlinkSync(src, dest);` throw if the symlink already exists?

Reply:
@springmeyer ya - `fs.symlinkSync` doesn't have an `overwrite` option, it just errors if a file already exists.